app/css/openmrs-addonmanager.css

```diff
@@ -611,3 +612,74 @@ body.loading .waiting-modal {
 .drop-zone-reject {
     border: 5px solid #ff8080;
 }
+
+#modules-table{
+    font-size: 0.95em;
+}
+
+.table-striped > tbody > tr:nth-child(2n+1) > td, .table-striped > tbody > tr:nth-child(2n+1) > th {
```

Comment:
Such complex selectors should be avoided it slows down the browser rendering process especially when a repaint or a style recalculation has to occur.

Reply:
`tr:nth-child(2n+1)`, this is the only calculation, and is being used to alternate the styling to every other element